bluebottle/bluebottle_drf2/serializers.py

```diff
@@ -40,10 +40,10 @@ def to_internal_value(self, data):
 
 class SorlImageField(RestrictedImageField):
     def __init__(self, geometry_string, **kwargs):
-        self.crop = kwargs.pop('crop', 'center')
```

Comment:
Yeah true. The original idea was that you could pass other imagick properties, but since we don't pass them  we could skip using this.kwargs